pkg/api/helpers.go

```diff
@@ -178,3 +179,53 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	}
 	return c
 }
+
+// GetAccessModesAsString returns a string representation of an array of access modes.
+// modes, when present, are always in the same order: RWO,ROX,RWX.
+func GetAccessModesAsString(modes []PersistentVolumeAccessMode) string {
+	modesAsString := ""
+
+	if contains(modes, ReadWriteOnce) {
+		appendAccessMode(&modesAsString, "RWO")
+	}
+	if contains(modes, ReadOnlyMany) {
+		appendAccessMode(&modesAsString, "ROX")
+	}
+	if contains(modes, ReadWriteMany) {
+		appendAccessMode(&modesAsString, "RWX")
+	}
+
+	return modesAsString
+}
+
+// GetAccessModesAsString returns an array of AccessModes from a string created by GetAccessModesAsString
+func GetAccessModesFromString(modes string) []PersistentVolumeAccessMode {
+	strmodes := strings.Split(modes, ",")
+	accessModes := []PersistentVolumeAccessMode{}
+	for _, s := range strmodes {
+		if s == "RWO" {
+			accessModes = append(accessModes, ReadWriteOnce)
+		} else if s == "ROX" {
+			accessModes = append(accessModes, ReadOnlyMany)
+		} else if s == "RWX" {
+			accessModes = append(accessModes, ReadWriteMany)
+		}
+	}
+	return accessModes
```

Comment:
This comment is not addressed.  Is it OK that "RWX,RWO,ROX" produces a different-but-equivalent slice as "ROX,RWX,RWO" ?  Is slice the right structure, or should it be a Set?